packages/web-components/src/design-tokens.ts

```diff
@@ -2003,25 +2051,37 @@ export const colorPaletteAnchorForeground2 = create<string>('colorPaletteAnchorF
  */
 export const colorPaletteAnchorBorderActive = create<string>('colorPaletteAnchorBorderActive').withDefault('#394146');
 
+// ----------------------------------------
+// Shadow
+// ----------------------------------------
+
 /**
  * @public
  */
-export const shadow2 = create<string>('shadow2').withDefault('0 0 2px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.14)');
+export const shadow2 = create<string>('shadow2').withDefault(
+  `0 0 2px ${colorNeutralShadowAmbient.$value}, 0 1px 2px ${colorNeutralShadowKey.$value}`,
```

Comment:
Added Fluent 2 spec'd colors to shadow tokens. They still need proper sizes (future).

This seems to work, but not sure if it is idiomatic. The shadow colors are defined by other color tokens. I used the `$value` property of those tokens to set the shadow color.

Reply:
Tokens work was merged in #4